pulpcore/cli/container/context.py

```diff
@@ -16,13 +18,28 @@ class PulpContainerDistributionContext(PulpEntityContext):
 
 
 class PulpContainerRemoteContext(PulpEntityContext):
-    ENTITY = "remote"
+    ENTITY = "container remote"
+    ENTITIES = "container remotes"
     HREF = "container_container_remote_href"
     LIST_ID = "remotes_container_container_list"
     CREATE_ID = "remotes_container_container_create"
     UPDATE_ID = "remotes_container_container_partial_update"
     DELETE_ID = "remotes_container_container_delete"
 
+    def preprocess_body(self, body: EntityDefinition) -> EntityDefinition:
+        body = super().preprocess_body(body)
+        for nullable in [
+            "ca_cert",
+            "client_cert",
+            "client_key",
+            "password",
+            "proxy_url",
+            "username",
+        ]:
```

Comment:
There might be a remote chance, we get the names of those beasts from the schema.